packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -25,22 +52,49 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
 }
 
 export const getDroppingStyles = (
-  targetEl: HTMLDivElement,
+  element: HTMLDivElement,
   bbox: ClientRect,
   maxTop: number,
-  timeRemaining: number
+  timeRemaining: number,
+  targetId?: string | null,
+  isClose?: boolean,
+  lastStyle?: React.CSSProperties
 ) => {
   const spotlightEl = document.getElementById('spotlight')
-  const showAboveSpotlight = spotlightEl ? spotlightEl.contains(targetEl) : false
+  const isTargetInSpotlight = targetId
+    ? spotlightEl && document.querySelector(`div[${DragAttribute.DROPPABLE}='${targetId}']`)
+    : false
+  const isBelongsToSpotlight = spotlightEl && spotlightEl.contains(element)
+  const showAboveSpotlight = isBelongsToSpotlight || isTargetInSpotlight
   const {top, left} = bbox
-  const minTop = getMinTop(top, targetEl)
+  const minTop = getMinTop(top, element)
   const clippedTop = Math.min(Math.max(minTop, top), maxTop - bbox.height)
   const isClipped = clippedTop !== top
+
+  const fadeInAnimation = `${reflectionSpotlightFadeIn.toString()} 0.5s linear 0s forwards`
+  const fadeOutAnimation = `${reflectionSpotlightFadeOut.toString()} 0.5s linear 0s forwards`
+  let animation: string | undefined = undefined
+
+  if (spotlightEl && !isBelongsToSpotlight && lastStyle) {
+    if (isTargetInSpotlight && !isClose) {
+      animation = fadeInAnimation
```

Comment:
Perhaps we could create two `const` variables, `isFadingIn` and `isFadingOut` instead of a `let` so it's easier to follow. For example, `const isFadingIn = isTargetInSpotlight && !isClose etc` and then on line 97 we could do `animation: isFadingIn ? fadeInAnimation : isFadingOut ? fadeOutAnimation : undefined`